app/controllers/OtherReasonController.scala

```diff
@@ -169,9 +170,9 @@ class OtherReasonController @Inject()(whenDidBecomeUnablePage: WhenDidBecomeUnab
     }
   }
 
-  def onPageLoadForUploadComplete(mode: Mode): Action[AnyContent] = (authorise andThen dataRequired) {
+  def onPageLoadForAnotherFileUpload(mode: Mode): Action[AnyContent] = (authorise andThen dataRequired) {
```

Comment:
I'm not sure why you have removed this? We have onPageLoadForUploadComplete as a placeholder so we can test that the logic is correct